File: JavaScript Fundamentals/Functions/loadingBar.js
function solve(num) {
    if (num == 100) console.log('100% Complete!');
    else {
        let percentCounter = num / 10;
        let dotCounter = 10 - percentCounter;
        let result = '['

        for (let i = 1; i <= percentCounter; i++) result += '%';
        for (let i = 1; i <= dotCounter; i++) result += '.';
        result += ']';
        console.log(`${num}% ${result}`);
        console.log('Still loading...');
    }
}

File: Python Fundamentals/Functions/loading_bar.py
def loading_bar(value):
    if int(value) == 100:
        print('100% Complete!')
        print('[%%%%%%%%%%]')
    else:
        percent_counter = int(value[:1])
        dot_counter = 10 - percent_counter
        result = '%' * percent_counter + '.' * dot_counter

        print(f'{value}% [{result}]')
        print('Still loading...')


number_inp = input()
loading_bar(number_inp)
